doc/src/build/cli-client.md

```diff
@@ -998,6 +998,9 @@ Please see the [Move developer documentation](move/index.md) for a
 description on how to [write a simple Move code package](move/write-package.md),
 which we can publish using Sui client's `publish` command.
 
+> **Important:** All calls to functions in the `debug` module must be removed from no-test code
```

Comment:
What do we mean by "no-test code" here? I see no other reference on this page.

Reply:
Added additional explanation in both places.

Reply:
> Added additional explanation in both places.

Thanks, Adam! LGTM.